tests/unit/networking.tcl

```diff
@@ -285,6 +285,46 @@ start_server {config "minimal.conf" tags {"external:skip"} overrides {enable-deb
             set info [r info stats]
             set new_prefetch_entries [getInfoProperty $info io_threaded_total_prefetch_entries]
             assert_equal $prefetch_entries $new_prefetch_entries
+      }
+
+      start_server {} {
+            test {replicas writes are offloaded to IO threads} {
+                set master [srv -1 client]
+                set master_host [srv -1 host]
+                set master_port [srv -1 port]
+    
+                set replica [srv 0 client]
+                $replica replicaof $master_host $master_port
+    
+                wait_for_condition 500 100 {
+                    [s 0 master_link_status] eq {up}
+                } else {
+                    fail "Replication not started."
+                }
+                
+                # get the current io_threaded_writes_processed
+                set info [$master info stats]
+                set io_threaded_writes_processed [getInfoProperty $info io_threaded_writes_processed]
+                
+                # Send a write command to the master
+                $master set a 1
```

Comment:
```suggestion
                # Send a write command to the primary
                $primary set a 1
```
We were supposed to move to inclusive language awhile ago, I realize there is a bunch of references to primary and slave now that we let sneak back in. Should probably do some cleanup.

Reply:
Thanks, changed.